dist/mjs/interface/interface.d.ts

```diff
@@ -257,6 +257,7 @@ export interface BaseAddImageParams {
     circuit_size: number;
     prove_payment_src: ProvePaymentSrc;
     auto_submit_network_ids: number[];
+    inherited_merkle_data_md5?: string;
```

Comment:
Better to be as the above prove_payment_src to have a type. (Backend rust should be Option<Text, not Text<Option